packages/application/src/domain/application/application.ts

```diff
@@ -460,39 +475,44 @@ rejectGovernanceReview(details: GovernanceReviewRejectedData) {
 
     this.applicantOrgName = event.file.organization || this.applicantOrgName
 
-    if(this.applicationStatus === ApplicationStatus.META_APPROVAL_PHASE){
-      this.allocationTooling = ["smart_contract_allocator"]
+    if (this.applicationStatus === ApplicationStatus.META_APPROVAL_PHASE) {
+      this.allocationTooling = ['smart_contract_allocator']
       this.pathway = 'MA'
       this.ma_address = '0xB6F5d279AEad97dFA45209F3E53969c2EF43C21d'
     }
-    if(this.applicationStatus === ApplicationStatus.RKH_APPROVAL_PHASE){
-      this.allocationTooling = [],
-      this.pathway = 'RKH',
-      this.ma_address = 'f080'
+    if (this.applicationStatus === ApplicationStatus.RKH_APPROVAL_PHASE) {
+      ;(this.allocationTooling = []), (this.pathway = 'RKH'), (this.ma_address = 'f080')
```

Comment:
I think the commas in the original code were a bug.

```suggestion
      this.allocationTooling = []
      this.pathway = 'RKH'
      this.ma_address = 'f080'
```